docs/source/modules/socrates.rst

```diff
@@ -0,0 +1,209 @@
+| Name                           | Default       | Description                                         |
++================================+===============+=====================================================+
+| ``socrates_hires_mode``        | False         | If false then run in 'GCM mode', and                |
+|                                |               | if true then usehigh-res spectral file              |
++--------------------------------+---------------+-----------------------------------------------------+
+| ``lw_hires_spectral_filename`` | 'unset'       |                                                     |
++--------------------------------+---------------+-----------------------------------------------------+
+| ``sw_hires_spectral_filename`` | 'unset'       |                                                     |
++--------------------------------+---------------+-----------------------------------------------------+
+| ``lw_spectral_filename``       | 'unset'       | Longwave spectral file, which can be found at       |
+|                                |               | Socrates source code ``data`` directory (e.g.       |
+|                                |               | ``data/spectra/ga7/sp_lw_ga7``) or self-generated   |
++--------------------------------+---------------+-----------------------------------------------------+
+| ``sw_spectral_filename``       | 'unset'       | Shortwave spectral file, which can be found at      |
+|                                |               | Socrates source code ``data`` directory (e.g.       |
+|                                |               | ``data/spectra/ga7/sp_sw_ga7``) or self-generated   |
++--------------------------------+---------------+-----------------------------------------------------+
+
+CO2, ozone and other gases
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+To include radiative effects of water vapor, CO2 and ozone, the following switches should be ``True``:
+
++-------------+---------------+-----------------------------------------------------+
+| Name        | Default       | Description                                         |
++=============+===============+=====================================================+
+| ``inc_h2o`` | True          | To include radiative effects of water vapor         |
++-------------+---------------+-----------------------------------------------------+
+| ``inc_co2`` | True          | To include radiative effects of CO2                 |
++-------------+---------------+-----------------------------------------------------+
+| ``inc_o3``  | True          | To include radiative effects of ozone               |
++-------------+---------------+-----------------------------------------------------+
+
+
+In addition, you need to set their concentrations by specifing them directly or reading from the external files (needs to pay attention to the concentration units):
```

Comment:
I'd change `needs to pay attention to the concentration units` to `close attention should be paid to the concentration units`